app/views/start/index.html.slim

```diff
@@ -1,5 +1,5 @@
 .govuk-grid-column-two-thirds
-  h1 class="govuk-heading-xl" = t(".efe_heading")
+  h1 class="govuk-heading-xl" = t("service.name")
```

Comment:
The ticket also says we need to update the header so that `gov.uk` on the left links to gov.uk.

Reply:
I know - but the standard doesn't seem to have it and the standard component (which we're using) doesn't produce it. Will be triple-checking with Jo

Reply:
You can make the standard component do what you want by passing an explicit `homepage_url` _and_  `service_url` I believe: https://github.com/DFE-Digital/govuk-components/blob/main/app/components/govuk_component/header_component.rb#L22

Reply:
Yes - but there's research that suggests it's confusing for users to take them back to gov.uk

Reply:
ooh, interesting that research is point in this new direction - I've always found it confusing that link to gov.uk. However since this link is not a core part of our service, I suggest we don't overthink it. On this basis, how about we simply defer to the convention of linking to gov.uk?

Reply:
@davidread yes I've gone with that in the end rather than digging the hole any deeper

Reply:
👍 I'll leave others to do the approval